client/payments/data/actions.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import { apiFetch } from '@wordpress/data-controls';
+
+/**
+ * Internal dependencies
+ */
+import { ACTION_TYPES as TYPES } from './action-types';
+import { API_NAMESPACE } from './constants';
+import { PaymentGateway, RestApiError, SelectorKeysWithActions } from './types';
+
+export function getPaymentGatewaysSuccess(
+	paymentGateways: Array< PaymentGateway >
```

Comment:
For array's it is preferred to use `PaymentGateway[]` instead of wrapping it in `Array<>`, it looks a little cleaner as well.
There is a few other spots that would have to be updated.

Reply:
Updated all changes in https://github.com/woocommerce/woocommerce-admin/commit/5687a413358e711567e00fd0579a87fcd9897d13